src/components/CommandBar.js

```diff
@@ -213,7 +213,7 @@ const CommandBar = React.createClass({
       () => toggleAllBreakpoints(!breakpointsDisabled),
       "toggleBreakpoints",
       breakpointsDisabled ? "breakpoints-disabled" : "",
-      "Disable Breakpoints"
+      breakpointsDisabled ? "Enable Breakpoints" : "Disable Breakpoints"
```

Comment:
Could we move this label into L10N stuff? @jasonLaster maybe a separate PR for that.